tests/queries/0_stateless/01600_count_of_parts_metrics.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+verify_sql="SELECT COUNT(1)
+FROM
+(SELECT
+    SUM(IF(metric = 'Parts', value, 0)) AS Parts,
+    SUM(IF(metric = 'PartsActive', value, 0)) AS PartsActive,
+    SUM(IF(metric = 'PartsInactive', value, 0)) AS PartsInactive
+FROM system.metrics) as a INNER JOIN
+(SELECT
+    toInt64(SUM(1)) AS Parts,
+    toInt64(SUM(IF(active = 1, 1, 0))) AS PartsActive,
+    toInt64(SUM(IF(active = 0, 1, 0))) AS PartsInactive
+FROM system.parts
+) as b USING (Parts,PartsActive,PartsInactive)"
+
+verify(){
+for _ in $(seq 1 10)
```

Comment:
The test does not passed after merge - https://clickhouse-test-reports.s3.yandex.net/0/f91626e7ff352d87283e7b8dafed94fd6ef38c8d/fast_test.html#fail1

@weeds085490 can you please take a look? Maybe 10 retries is not enough?